src/main/java/teammates/ui/pagedata/StudentHomePageData.java

```diff
@@ -71,6 +71,7 @@ private void setCourseTables(List<CourseDetailsBundle> courses,
                     getStudentSubmissionStatusForSession(feedbackSession, hasSubmitted),
                     getStudentPublishedStatusForSession(feedbackSession),
                     TimeHelper.formatTime12H(feedbackSession.getEndTimeLocal()),
```

Comment:
This isn't directly related, but since you are improving the student home page by adding the sort by deadline, could you take this chance to also show the time zone info in the deadline display? Simply change `formatTime12H` here to `formatDateTimeForSessions` and run godmode. Forgot to do this in #8687 😅